cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go

```diff
@@ -128,25 +130,29 @@ type autoscalingGceClientV1 struct {
 	projectId string
 	domainUrl string
 
-	// Can be overridden, e.g. for testing.
-	operationWaitTimeout  time.Duration
+	// Amount of time to wait for operation before terminating
+	operationWaitTimeout time.Duration
+	// Time interval between wait calls
 	operationPollInterval time.Duration
+	// Amount of time to start async operation, maps 1:1 to http calls
```

Comment:
nit: "Amount of time to start async operation" sounds like sleep before doing something in a separate goroutine, not making an external API call. Maybe just skip this comment?

Reply:
Yeah, it may be slightly confusing, `operationPerCallTimeout` should be clear enough to remove the comment, done